src/lib/modernExtend.ts

```diff
@@ -1603,10 +1604,40 @@ export function electricityMeter(args?: ElectricityMeterArgs): ModernExtend {
         haElectricalMeasurement: {
             // Report change with every 5W change
             power: {attribute: 'activePower', divisor: 'acPowerDivisor', multiplier: 'acPowerMultiplier', forced: args.power, change: 5},
+            power_phase_b: {attribute: 'activePowerPhB', divisor: 'acPowerDivisor', multiplier: 'acPowerMultiplier', forced: args.power, change: 5},
+            power_phase_c: {attribute: 'activePowerPhC', divisor: 'acPowerDivisor', multiplier: 'acPowerMultiplier', forced: args.power, change: 5},
```

Comment:
Is it possible to add a new property that sums power, power_phase_b and power_phase_c and adds it as an exposed property? How could such a feature be implemented?

Reply:
If supported, via the "totalActivePower" attribute?